### PR TITLE
Update ERC-1620 to "Last Call"

### PR DESCRIPTION
**Pull request by PaulRBerg**
As per the announcement in https://github.com/ethereum/EIPs/issues/1620, update the markdown version of ERC-1620 to its final version from Sep 27, 2019.

### CODE REVIEW

File: EIPS/eip-1620.md [hunk 1]
```diff
@@ -1,305 +1,198 @@
----
+```
 eip: 1620
 title: ERC-1620 Money Streaming
-author: Paul Berg (@PaulRBerg) <hello@paulrberg.com>
+author: Paul Razvan Berg (@PaulRBerg) <paul@sablier.finance>
 discussions-to: https://github.com/ethereum/EIPs/issues/1620
-status: Draft
+status: Final
```
Comment (axic) [inline EIPS/eip-1620.md]: One cannot just set an EIP to Final from Draft. It must go through the Last Call process. Please check EIP-1.

Comment (PaulRBerg) [reply EIPS/eip-1620.md]: Epic fail on my end. I updated the status to "Last Call" in September, but I forgot to submit a PR for that.

**Comment by PaulRBerg** [comment]
Does this work now @axic?

File: EIPS/eip-1620.md [hunk 2]
```diff
@@ -1,305 +1,198 @@
 ---
 eip: 1620
 title: ERC-1620 Money Streaming
-author: Paul Berg (@PaulRBerg) <hello@paulrberg.com>
+author: Paul Razvan Berg (@PaulRBerg) <paul@sablier.finance>
 discussions-to: https://github.com/ethereum/EIPs/issues/1620
-status: Draft
+status: Last Call
```
Comment (axic) [inline EIPS/eip-1620.md]: Last call requires a `review-period-end` field. It is customary to provide at least 2 weeks for it, but given it is holiday season, I would recommend to go with at least 3-4 weeks.

File: EIPS/eip-1620.md [hunk 3]
```diff
@@ -1,305 +1,198 @@
...
-    - `balance`: the total funds left in the stream
-    - `timeframe`: as defined below
-    - `rate`: as defined below
-
-```solidity
-  struct Stream {
-    address sender;
+The structure of a stream should be as follows:
+
+  - `Stream`
```
Comment (axic) [inline EIPS/eip-1620.md]: These seem to be ended on the same level?

File: EIPS/eip-1620.md [hunk 4]
```diff
@@ -1,305 +1,198 @@
...
+[Sablier](https://sablier.finance) is a champion project for this ERC. It is what we call the protocol for real-time finance.
 
 ## Specification
-<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
 
+<!--The technical specification should describe the syntax and semantics of any new feature. The specification should be detailed enough to allow competing, interoperable implementations for any of the current Ethereum platforms (go-ethereum, parity, cpp-ethereum, ethereumj, ethereumjs, and [others](https://github.com/ethereum/wiki/wiki/Clients)).-->
```
Comment (axic) [inline EIPS/eip-1620.md]: These comments are not needed.

File: EIPS/eip-1620.md [hunk 5]
```diff
@@ -1,305 +1,198 @@
...
-#### withdraw
-
-Withdraws all or a fraction of the available funds.
+#### createStream
 
-MUST allow only the recipient to perform this action.
+Creates a new stream funded by `msg.sender` and paid towards `recipient`. MUST throw if `recipient` is the zero address, the contract itself or the caller themselves. MUST throw if `startTime` is before the current block timestamp. MUST throw if `stopTime` is before `startTime`. MUST throw if `deposit` is not a multiple of the time delta. MUST throw if the contract is not allowed to transfer enough tokens.
 
-**Triggers Event**: [LogWithdraw](#logwithdraw)
+**Triggers Event**: [CreateStream](#create-stream)
```
Comment (axic) [inline EIPS/eip-1620.md]: Build is failing because these links don't seem to work.

**Comment by PaulRBerg** [comment]
Thank you for your help @axic in getting this properly formatted. Just pushed a commit, everything should be good now.

I set the review period to end on December 11, 2020.

**Comment by eip-automerger** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):

 - Trying to change EIP 1620 state from Draft to Last Call

**Comment by PaulRBerg** [comment]
Hey! Is there anything else I need to do to get this merged?

**Comment by PaulRBerg** [comment]
Sorry to follow up, but people are making references to the old version: https://twitter.com/3esmit/status/1216111454509830144

Would love to get this in the "Last Call" status!

**Comment by PaulRBerg** [comment]
Closing this in favour of https://github.com/ethereum/EIPs/pull/2467

**Pull request closed without merging**